openpgp-dsm/src/lib.rs

```diff
@@ -1855,8 +1856,17 @@ fn try_unlock_p12(cert_file: String) -> Result<Identity> {
     // Try to unlock certificate without password first
     let mut first = true;
     if let Ok(id) = Identity::from_pkcs12(&cert, "") {
-        Ok(id)
+        return Ok(id)
     } else {
+        // Try to unlock with env var passphrase
+        if let Ok(pass) = env::var(ENV_P12_PASS) {
+            if let Ok(id) = Identity::from_pkcs12(&cert, &pass) {
+                info!("unlocked PKCS12 identity with {:?}", ENV_P12_PASS);
+                return Ok(id)
+            } else {
+                info!("could not unlock PKCS12 with {:?}", ENV_P12_PASS);
```

Comment:
Nit of nits (again, feel free to ignore): "PKCS12 identity"?

Reply:
Yes